map_gui/src/render/road.rs

```diff
@@ -91,24 +91,22 @@ impl DrawRoad {
                     app.cs()
                         .zoomed_road_surface(LaneType::Driving, r.get_rank())
                 };
-
-                if false {
```

Comment:
I would vote to merge this PR, but keep the behavior disabled like this. I don't want to cut over until the results are legible, and right now we have:
![Screenshot from 2022-03-24 10-07-21](https://user-images.githubusercontent.com/1664407/159894799-3aabc62c-7b64-48e8-9541-f6071d28feb8.png)
![Screenshot from 2022-03-24 10-07-14](https://user-images.githubusercontent.com/1664407/159894805-bf2a0e5d-9d13-44df-93d2-fdaaae85ae2d.png)

We could also work in a branch, but that feels like more hassle than a hardcoded "experiment flag" like this

Reply:
Is your complaint about the text being overlaid over the center line, or about the letter spacing?

But yes, +1 to merging this and fixing niggles in a subsequent PR

Reply:
Letter spacing - James already plans to look at hiding the center-line next. In the first example, "iv" in "Drive" looks weird. The spacing in "Miller" looks odd too. This probably comes down to how SVG renders text along a curved path and isn't something we can fix. Maybe we let the user enable/disable curved labels if the default we pick doesn't work well for them.

Reply:
Huh, looks like you're right (Here Dragons Abound had the same problems with curved labels and SVG: https://heredragonsabound.blogspot.com/2017/06/path-labels-part-four.html and just ended up using gently curved circular arcs: https://heredragonsabound.blogspot.com/2017/09/labeling-coast-part-two.html